certbot/plugins/webroot_test.py

```diff
@@ -274,7 +273,7 @@ def setUp(self):
 
     def test_webroot_map_action(self):
         args = self.parser.parse_args(
-            ["--webroot-map", '{{"thing.com":"{0}"}}'.format(self.path)])
+            ["--webroot-map", '{{"thing.com":"{0}"}}'.format(self.path).replace('\\', '\\\\')])
```

Comment:
A big mess, an escape-ception. You want a string, containing backslashes (because Windows), that is a field in a serialized JSON (so a string). So you need to add a backslash to the backslash to escape it in the string field, and again a backslash for each backslash to escape then in the JSON string.

Ok its ugly, I was tired of theses problems because Windows decided to use backslashes for their path. I correct it with a more elegant json.dumps.

Reply:
Oh jeez. I didn't realize we also had to escape it for the JSON string.

The `json.dumps` solution works nicely though!